system/src/system_network_internal.h

```diff
@@ -57,15 +57,15 @@ extern uint32_t wlan_watchdog_base;
 inline void ARM_WLAN_WD(uint32_t x) {
     wlan_watchdog_base = HAL_Timer_Get_Milli_Seconds();
     wlan_watchdog_duration = x;
-    WAN_WD_DEBUG("WD Set %d",(x));
+    WAN_WD_DEBUG("WD Set %d\r\n",(x));
```

Comment:
Oh, well I had changed it to use the new logging.h API and made this change: `LOG_PRINTF_C(INFO, "system.wdt", x, ##__VA_ARGS__)` to `DEBUG(x,__VA_ARGS__)`.  I noticed the lines weren't correct so I added the newlines.  But then I backed out those changes and thought it might have always been that way since we never enable it.  Should I make the new logging API change, since we don't typically compile with DEBUG_WAN_WD it should be removed anyway.

Reply:
I'd remove them for now and make the logging changes in a separate PR. We have a relevant CH story.

Reply:
Will do.